packages/amplify-category-function/provider-utils/awscloudformation/function-template-dir/index.js.ejs

```diff
@@ -1,8 +1,10 @@
 <%= props.topLevelComment %>
 
-exports.handler = function (event, context) { //eslint-disable-line
-  console.log(`value1 = ${event.key1}`);
-  console.log(`value2 = ${event.key2}`);
-  console.log(`value3 = ${event.key3}`);
-  context.done(null, 'Hello World'); // SUCCESS with message
```

Comment:
Won't be still need context.done even if we use arrow functions? or does returning the response replaces the need for having context.done?